File: DesignPatternsJava/src/com/facade/Behavior.java
package com.facade;

public class Behavior {
	private Student student;
	private String behavior = "Very Polite";
	public Behavior(Student aStudent){
		student=aStudent;
	}
	public void getStudentBehavior(){
		System.out.println(student.getName()+"'s behavior is "+behavior);
	}
}

File: DesignPatternsDotNet/Facade/Behavior.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Facade
{
    public class Behavior
    {
        private Student student;
        private string behavior = "Very Polite";
        public Behavior(Student aStudent)
        {
            student = aStudent;
        }
        public void GetStudentBehavior(){
		Console.WriteLine(student.Name+"'s behavior is "+behavior);
	}
    }
}
